src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -68,6 +68,24 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       });
     }
 
+    // When rendering in Kibana Platform mode, only serve the assets needed by Core.
+    if ('{{appId}}' === 'core') {
+      load([
+          {{#each sharedJsDepFilenames}}
+            '{{../regularBundlePath}}/kbn-ui-shared-deps/{{this}}',
+          {{/each}}
+          '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedJsFilename}}',
+          '{{regularBundlePath}}/plugin/kibanaUtils/kibanaUtils.plugin.js',
+          '{{regularBundlePath}}/plugin/esUiShared/esUiShared.plugin.js',
+          '{{regularBundlePath}}/plugin/kibanaReact/kibanaReact.plugin.js',
+          '{{regularBundlePath}}/core/core.entry.js',
+          {{#each styleSheetPaths}}
```

Comment:
Seems like we don't need to load all the styleSheetPaths for new platform plugins either.

Reply:
Good catch. With some shuffling of styles out of legacy and into Core, I was able to drop most of these but some are still needed (styles from kbn-ui-shared-deps).